### PR TITLE
DOC: Adding more documentation on how to run rhapsody 

### PR DESCRIPTION
**Pull request by mortonjt**
This also provides some details on how to interpret the results

CC @ElDeveloper  - this may help with running / interpreting the model

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,3 +93,98 @@ qiime emperor biplot \
...
+**A** : Both `epochs` and `batch-size` contribute to determining how long the algorithm will run, namely
+
+**Number of iterations = `epoch #` multiplied by the ( Total # of microbial reads / `batch-size` parameter)**
+
+This also depends on if your program will converge. The `learning-rate` specifies the resolution (smaller step size = smaller resolution, but may take longer to converge). You will need to consult with Tensorboard to make sure that your model fit is sane. See this paper for more details on gradient descent: https://arxiv.org/abs/1609.04747
+
+If you are running this on a CPU, 16 cores, a run that reaches convergence should take about 1 day.
+If you have a GPU - you maybe able to get this down to a few hours.  However, some finetuning of the `batch_size` parameter maybe required -- instead of having a small `batch_size` < 100, you'll want to bump up the `batch_size` to between 1000 and 10000 to fully leverage the speedups available on the GPU.
+
+Credits to Lisa Marotz ([@lisa55asil](https://github.com/lisa55asil)),  Yoshiki Vasquez-Baeza ([@ElDeveloper](https://github.com/ElDeveloper)) and Julia Gauglitz for their README contributions.
```
Comment (ElDeveloper) [inline README.md]: Thanks for the inclusion here :) should be Vazquez with a z instead of Vasquez :P

Comment (mortonjt) [reply README.md]: jesus 😆. fixed.

**Comment by mortonjt** [comment]
Ok - I think the comments on this have been addressed.  Merging this in.  Thanks @ElDeveloper !

**Comment by ElDeveloper** [comment]
👍

**Pull request merged**